src/chart/graph/GraphSeries.ts

```diff
@@ -229,6 +232,28 @@ export interface GraphSeriesOption
      * auto curveness for multiple edge, invalid when `lineStyle.curveness` is set
      */
     autoCurveness?: boolean | number | number[]
+
+    thumbnail?: {
+        show?: boolean,
+
+        top?: number | string,
+
+        bottom?: number | string,
+
+        left?: number | string,
+
+        right?: number | string,
+
+        scale?: number,
+
+        borderColor?: Color,
+
+        backgroundColor?: Color,
+
+        overlayBackgroundColor?: Color,
+
+        selectedDataBackground?: AreaStyleProps
```

Comment:
Wrong type here. `AreaStyleProps` is the key list. It should be `ItemStyle`. Also I will prefer the name `selectedBackgroundStyle`